dev/com.ibm.etools.maven.liberty.integration/src/com/ibm/etools/maven/liberty/integration/manager/internal/LibertyMavenRuntimeProvider.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.etools.maven.liberty.integration.manager.internal;
+
+import org.eclipse.wst.server.core.IServer;
+
+import com.ibm.etools.maven.liberty.integration.internal.LibertyMaven;
+import com.ibm.ws.st.liberty.buildplugin.integration.internal.AbstractLibertyRuntimeProvider;
+import com.ibm.ws.st.liberty.buildplugin.integration.internal.ILibertyBuildPluginImpl;
+
+public class LibertyMavenRuntimeProvider extends AbstractLibertyRuntimeProvider {
+
+    public LibertyMavenRuntimeProvider() {
+        // Empty
+    }
+
+    // Maven specific
+
+    /** {@inheritDoc} */
+    @Override
+    protected ILibertyBuildPluginImpl getBuildPlugin() {
+        return LibertyManager.getInstance().getBuildPluginImpl();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    protected String getTargetServerConfig(String serverName) {
+        return "target/liberty/wlp/usr/servers/" + serverName + "/server.xml";
```

Comment:
Will this work if installDir is set by the user?

Reply:
I tried to use installDir in the build.gradle file, and it didn't work for me.  Supposedly, you can point to a pre-installed Liberty, so this could be outside of the Gradle/Maven project, so therefore they are Workspace resources?  So, I could not get this to work.  We could document this as a limitation.

Reply:
@eharris369 , I had opened ci.gradle issue 97.    We need an updated build.

Reply:
@eharris369 , I've updated the runtime provider code so that it uses serverOutputDirectory instead.  Please review.